seqr/utils/search/add_data_utils.py

```diff
@@ -124,8 +124,8 @@ def prepare_data_loading_request(projects: list[Project], sample_type: str, data
         'projects_to_run': project_guids,
         'callset_path': data_path,
         'sample_type': sample_type,
-        'dataset_type': _dag_dataset_type(sample_type, dataset_type),
-        'reference_genome': GENOME_VERSION_LOOKUP[genome_version],
+        'dataset_type': dag_dataset_type(dataset_type, sample_type),
+        'reference_genome': reference_genome_version(genome_version),
     }
```

Comment:
rather than exporting the formatting for these variables, it might be better to have a `format_loading_pipeline_variables` helper  that could be used both here and in `trigger_airflow_delete_families` - something like 
```
def format_loading_pipeline_variables(projects: list[Project], sample_type: str, dataset_type: str, genome_version: str, **kwargs):
    project_guids = sorted([p.guid for p in projects])
    variables = {
        'projects_to_run': project_guids,
        'sample_type': sample_type,
        'dataset_type': _dag_dataset_type(sample_type, dataset_type),
        'reference_genome': GENOME_VERSION_LOOKUP[genome_version],
        **kwargs
    }
```